Tests/CaptureCocoaLumberjack/CaptureCocoaLumberjackIntegrationTests.swift

```diff
@@ -0,0 +1,9 @@
+import CaptureCocoaLumberjack
+import CocoaLumberjackSwift
+import XCTest
+
+final class CaptureCocoaLumberjackIntegrationTests: XCTestCase {
+    func testAddingCaptureDDLogger() {
+        DDLog.add(CaptureDDLogger())
```

Comment:
what do these tests assert?

Reply:
@Augustyniak  it's more like a "standard" suggested on [the maven docs](https://docs.gradle.org/current/userguide/publishing_maven.html#publishing_maven:complete_example). but happy to change it, can you suggest here what would be your ideal path?